dashboards/Node_Summary.json

```diff
@@ -829,7 +829,7 @@
             "tableColumn": "",
             "targets": [
                 {
-                    "expr": "(node_memory_MemAvailable_bytes{node_name=~\"$host\"} or (node_memory_MemFree_bytes{node_name=~\"$host\"} + node_memory_Buffers_bytes{node_name=~\"$host\"} + node_memory_Cached_bytes{node_name=~\"$host\"})) / node_memory_MemTotal_bytes{node_name=~\"$host\"} * 100",
+                    "expr": "((node_memory_MemAvailable_bytes{node_name=~\"$host\"} or (node_memory_MemFree_bytes{node_name=~\"$host\"} + node_memory_Buffers_bytes{node_name=~\"$host\"} + node_memory_Cached_bytes{node_name=~\"$host\"})) / node_memory_MemTotal_bytes{node_name=~\"$host\"} or\n(rdsosmetrics_memory_free{node_name=~\"$host\"} + rdsosmetrics_memory_buffers{node_name=~\"$host\"} + rdsosmetrics_memory_cached{node_name=~\"$host\"})/ rdsosmetrics_memory_total{node_name=~\"$host\"}) * 100",
```

Comment:
rds_exporter should return metrics as node_exporter. Please rise issues instead of adding more hacks to the dashboards.